Agoda.Frameworks.DB/DbRepository_New.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Agoda.Frameworks.LoadBalancing;
+using Dapper;
+
+
+namespace Agoda.Frameworks.DB
+{
+    public partial class DbRepository // new simplified stuff
+    {
+        public async Task<object> ExecuteScalarAsync(
+            string dbName,
+            string SqlCommandString,
```

Comment:
Can we have unit tests for the new methods? If for nothing else, then to demonstrate how they are supposed to be used.

Reply:
yes, PR isnt 100% finished yet, just wanted to get a conversation going

Reply:
Done by #20